--- a/Makefile
+++ b/Makefile
@@ -687,8 +687,8 @@
 
 $(TIMESTAMP_FILE):
 		@mkdir -p $(dir $(TIMESTAMP_FILE))
-		@LC_ALL=C date +'#define U_BOOT_DATE "%b %d %C%y"' > $@.tmp
-		@LC_ALL=C date +'#define U_BOOT_TIME "%T"' >> $@.tmp
+		@LC_ALL=C date -u -d "@$(SOURCE_DATE_EPOCH)" +'#define U_BOOT_DATE "%b %d %C%y"' > $@.tmp
+		@LC_ALL=C date -u -d "@$(SOURCE_DATE_EPOCH)" +'#define U_BOOT_TIME "%T"' >> $@.tmp
 		@cmp -s $@ $@.tmp && rm -f $@.tmp || mv -f $@.tmp $@
 
 easylogo env gdb:
